test/rauc.t

```diff
@@ -1297,6 +1297,7 @@ test_expect_success ROOT,!SERVICE "rauc install (no service)" "
   test ! -s ${SHARNESS_TEST_DIRECTORY}/images/rootfs-1 &&
   rauc \
     --conf=${SHARNESS_TEST_DIRECTORY}/minimal-test.conf \
+    --override-boot-slot=system0 \
```

Comment:
With this change, do we still have tests for parsing the cmdline?

Reply:
We have a few explicit ones for `/dev/nfs` and `rauc.external` in `test/context.c`. Maybe these should be extended.

In sharness tests we don't have any as far as I see.

The ones that I have changed were more accidentally than intentionally testing /proc/cmdline.
The alternative would be to add explicit tests that depend on running in qemu or to expect all tests to run in qemu and remove all override-boot-slots.

Reply:
I've understood this as a request to add more tests and provided them now.